group_vars/fender-cluster/vars.yml

```diff
@@ -94,6 +94,12 @@ local_regular_users:
     groups: ['users', 'depad','umcg-atd', 'solve-rd']
   - user: 'ljohansson'
     groups: ['users', 'depad','umcg-atd', 'solve-rd']
+  - user: 'ddanis'
```

Comment:
These collaborators should be added to the ```solve-rd``` group, but adding them to ```depad``` and ```umcg-atd``` groups makes less sense to me...

Reply:
fixed